docs/content/en/docs/tasks/cluster/manual-renew-certs.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Manual Renew Certificates"
+linkTitle: "Manual Renew Certificates"
+weight: 31
+date: 2021-11-04
+description: >
+  How to renew certificates in etcd and control plane machines
+---
+
+Certificates in external etcd and control plane nodes expire after 1 year. The following table shows all the certificate files:
+
+
+| etcd node             | control plane node       |
+|-----------------------|--------------------------|
+| apiserver-etcd-client | apiserver-etcd-client    |
+| ca                    | ca                       |
+| etcdctl-etcd-client   | front-proxy-ca           |
+| peer                  | sa                       |
+| server                | etcd/ca.crt              |
+|                       | apiserver-kubelet-client |
+|                       | apiserver                |
+|                       | front-proxy-client       |
+
+You can renew all certificates in the table except ca by following the steps given below.
+
+### External etcd node
+
+If the cluster has external etcd, you need to renew the certificates in etcd nodes first.
+
+1. ssh into the etcd node and run the following commands:
+
+{{< tabpane >}}
+{{< tab header="Ubuntu or RHEL" lang="bash" >}}
+# backup certs
+cd /etc/etcd
+sudo mv pki pki.bak
+sudo mkdir pki
+sudo mv pki.bak/ca.* pki
+
+# run certificates join phase to regenerate the deleted certificates
+sudo etcdadm join phase certificates http://dumburl
+
+# etcd will auto pick the new certificates
+{{< /tab >}}
+
+{{< tab header="Bottlerocket" lang="bash" >}}
+# get the image name of the kubeadm-bootstrap container
+apiclient get | jq '.settings["host-containers"]["kubeadm-bootstrap"].source'
+
+# go to root shell from control container
+sudo sheltie
+
+# pull the image
+IMAGE_ID=${the image you got from previous step}
+ctr image pull ${IMAGE_ID}
+
+# backup certs
+cd /var/lib/etcd
+mv pki pki.bak
+mkdir pki
+mv pki.bak/ca.* pki
+
+
+# recreate certificates
+ctr run --mount type=bind,src=/var/lib/etcd/pki,dst=/etc/etcd/pki,options=rbind:rw --net-host --rm \
+${IMAGE_ID} tmp-cert-renew \
+/opt/bin/etcdadm join phase certificates http://dumburl --init-system kubelet
+
+# etcd will auto pick the new certificates
+{{< /tab >}}
+{{< /tabpane >}}
+
+2. Verify your etcd node is running correctly
+
+{{< tabpane >}}
+{{< tab header="Ubuntu or RHEL" lang="bash" >}}
+sudo etcdctl --cacert=ca.crt --cert=etcdctl-etcd-client.crt --key=etcdctl-etcd-client.key member list
```

Comment:
can we have full paths for crts and keys or say user needs to be in /etc/etcd/pki

Reply:
sure